### ISSUE TITLE
Upgrade symfony/console to v3

### ISSUE THREAD

**Issue by codeaid** [browscap/browscap-php#144]
I have a suggestion to upgrade Symfony console to the latest version, which at the moment is v3.0.1. I am using other libraries that require 3.* and am having problems using them together with browscap-php.

I tried upgrading it locally and the unit tests still pass. I don't think there are any breaking changes in the current version, hence would be nice to move away from the "legacy" version.

Run unit tests on the current 3.0.0-rc.1:
```
$ vendor/bin/phpunit 
PHPUnit 5.2.9 by Sebastian Bergmann and contributors.

...............................[2016-02-23 19:38:48] initializing converting process {"memory_peak_usage":"20 MB"}
[2016-02-23 19:38:48] started converting local file {"memory_peak_usage":"20 MB"}
.S.S...SSSSSSSSSSSSSSSSSSSSSSSSS  63 / 130 ( 48%)
SSSSSSSSSSSSSSSSS........................................S..... 126 / 130 ( 96%)
....                                                            130 / 130 (100%)

Time: 679 ms, Memory: 22.00Mb

OK, but incomplete, skipped, or risky tests!
Tests: 1770, Assertions: 177, Skipped: 45.
```

Update Symfony console:
```bash
$ vi composer.json
```

Apply the following patch:
```diff
diff --git a/composer.json b/composer.json
index 3ea0dea..1fc0a06 100644
--- a/composer.json
+++ b/composer.json
@@ -29,7 +29,7 @@
         "mimmi20/file-loader": "~2.0",
         "symfony/filesystem": "*",
         "symfony/finder": "~2.6",
-        "symfony/console": "~2.6",
+        "symfony/console": "^3.0",
         "monolog/monolog": "1.*"
     },
     "require-dev": {
```

Update composer:
```bash
$ composer update symfony/console
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing symfony/console (v2.8.2)
  - Installing symfony/console (v3.0.2)
    Downloading: 100%         

Writing lock file
Generating autoload files
```

Run unit tests on the updated dependencies:
```
$ vendor/bin/phpunit 
PHPUnit 5.2.9 by Sebastian Bergmann and contributors.

...............................[2016-02-23 19:39:37] initializing converting process {"memory_peak_usage":"20 MB"}
[2016-02-23 19:39:37] started converting local file {"memory_peak_usage":"20 MB"}
.S.S...SSSSSSSSSSSSSSSSSSSSSSSSS  63 / 130 ( 48%)
SSSSSSSSSSSSSSSSS........................................S..... 126 / 130 ( 96%)
....                                                            130 / 130 (100%)

Time: 668 ms, Memory: 22.00Mb

OK, but incomplete, skipped, or risky tests!
Tests: 1770, Assertions: 177, Skipped: 45.
```

**Comment by codeaid** [comment]
I opened a pull request to save you some time if you decide to go forward with the change.

**Comment by asgrim** [comment]
Thank you for reporting and checking so thoroughly! Happy with the change in general.

**Comment by asgrim** [comment]
Closed by #145
